clusterloader2/testing/density/rc.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    name: {{.Basename}}
+  name: {{.Basename}}
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    name: {{.Basename}}
+  template:
+    metadata:
+      labels:
+        name: {{.Basename}}
+        phase: phase1
+    spec:
+      containers:
+      - image: gcr.io/kubernetes-e2e-test-images/serve-hostname-amd64:1.0
```

Comment:
Maybe change this to pause pod (like what we use in our standard density test)?  That should be `k8s.gcr.io/pause:3.1`